### PR TITLE
Added documentation for gogogate2

### PR DESCRIPTION
**Pull request by dlbroadfoot**
**Description:**

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#13467

## Checklist:

- [x] Branch: Fixes, changes and adjustments should be created against `current`. New documentation for platforms/components and features should go to `next`.
- [x] The documentation follow the [standards][standards].

[standards]: https://home-assistant.io/developers/documentation/standards/

### CODE REVIEW

File: source/_components/cover.gogogate2.markdown [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+```yaml
+# Example configuration.yml entry
+cover:
+  - platform: gogogate2
+    username: email@email.com
+    password: password
+    ip_address: 192.168.1.200
+```
+
+Configuration variables:
```
Comment (frenck) [inline source/_components/cover.gogogate2.markdown]: Please use the configuration tags, for more information please see:
https://home-assistant.io/developers/documentation/create_page/#configuration

File: source/_components/cover.gogogate2.markdown [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+- **username** (*Required*): Your Gogogate2 account username.
+- **password** (*Required*): Your Gogogate2 account password.
+- **ip_address** (*Required*): IP address of your Gogogate2 device
```
Comment (frenck) [inline source/_components/cover.gogogate2.markdown]: The optional `api_key` configuration variable is missing in the docs but is present in the code.

**Review by frenck — CHANGES_REQUESTED** [review]

**Comment by frenck** [comment]
Thank you for this PR, @dlbroadfoot!

I've left you some comments to take a look at 👍

**Review by frenck — DISMISSED** [review]

**Comment by frenck** [comment]
Thanks, @dlbroadfoot for updating the PR! 🥇 

Looks good and can be merged as soon as the parent PR gets merged.

**Review by fabaff — DISMISSED** [review]

**Review by fabaff — APPROVED** [review]

**Pull request merged**